src/components/date_picker/super_date_picker/super_update_button.js

```diff
@@ -12,10 +12,16 @@ export class EuiSuperUpdateButton extends Component {
     isLoading: PropTypes.bool,
     isDisabled: PropTypes.bool,
     onClick: PropTypes.func.isRequired,
+
     /**
      * Passes props to `EuiToolTip`
      */
     toolTipProps: PropTypes.object,
+
+    /**
+     * Check if the popover is closed to display toggle
+     */
+    isPopoverClosed: PropTypes.bool,
```

Comment:
Can we rename to something more specific to `EuiSuperUpdateButton`? What the component ends up doing with this props is allow showing of the tooltip. My suggestion is
```suggestion
    showTooltip: PropTypes.bool,
```

Reply:
Can you also add a test for this new prop in this file https://github.com/elastic/eui/blob/ab24518fd0db319a1cc8db957966597eeae838ad/src/components/date_picker/super_date_picker/super_update_button.test.js

Reply:
@cchaos Thanks for the review. I have updated the props name and added a new test.